src/content/api/contextmodulefactory-hooks.md

```diff
@@ -0,0 +1,55 @@
+---
+title: ContextModuleFactory Hooks
+group: Plugins
+sort: 11
+contributors:
+  - iguessitsokay
+---
+
+The `ContextModuleFactory` module is used by the `Compiler` to generate dependencies from webpack specific [require.context](/api/module-methods/#requirecontext) API. It resolves the requested directory, generates requests for each file and filters against passed regExp. Matching dependencies then passes through [NormalModuleFactory](/api/normalmodulefactory-hooks).
+
+The `ContextModuleFactory` class extends `Tapable` and provides the following
+lifecycle hooks. They can be tapped the same way as compiler hooks:
+
+``` js
+ContextModuleFactory.hooks.someHook.tap(/* ... */);
+```
+
+As with the `compiler`, `tapAsync` and `tapPromise` may also be available
+depending on the type of hook.
+
+
+### `beforeResolve`
```

Comment:
```suggestion
### beforeResolve
```
Let's remove inline code style from heading as it's sort of disturbing when applied on headings. So far there're many usages on this site, but I'm going to remove them in the future.